bin/transform_bulk

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
Please name `transform-bulk`, for consistency with Transformer PSC, SK, DK post-refactor.